corehq/apps/cloudcare/templates/formplayer/dependencies.html

```diff
@@ -11,8 +11,8 @@
 <script src="{% static 'bootstrap-switch/dist/js/bootstrap-switch.js' %}"></script>
 <script src="{% static 'datetimepicker/build/jquery.datetimepicker.full.js' %}"></script>
 <script src="{% static 'cloudcare/js/util.js' %}"></script>
-<script src="{% static 'Caret.js/dist/jquery.caret.min.js' %}"></script>
-<script src="{% static 'At.js/dist/js/jquery.atwho.min.js' %}"></script>
+<script src="{% static 'Caret.js/dist/jquery.caret.js' %}"></script>
+<script src="{% static 'At.js/dist/js/jquery.atwho.js' %}"></script>
```

Comment:
had to do a fair bit of debugging these libraries to get atwho to work correctly in an iframe. these will get compressed later anyways